cmd/servicelog/list.go

```diff
@@ -90,7 +90,7 @@ func init() {
 	listCmd.Flags().BoolVarP(&serviceLogListInternalOnlyFlag, "internal", "i", serviceLogListInternalOnlyFlag, "Toggle if we should see internal messages")
 }
 
-func CreateListSLRequest(ocmClient *sdk.Connection, clusterId string, allMessages bool, internalMessages bool) *sdk.Request {
+func CreateListSLRequest(ocmClient *sdk.Connection, clusterId string, clusterUuid string, allMessages bool, internalMessages bool) *sdk.Request {
```

Comment:
The logic is good, but what do you think about this function having a `cluster *cmv1.Cluster` input instead of `clusterId` and `clusterUuid` individually?

To be fair, `CreateListSLRequest` probably won't be reused much, so I'm happy to get this merged without this change, but I think it would be clearer if we need to consume this function later.

Reply:
Agreed, that makes sense. That change has been made, thanks!